modules/nf-core/openms/idmerger/tests/main.nf.test

```diff
@@ -0,0 +1,66 @@
+nextflow_process {
+
+    name "Test Process OPENMS_IDMERGER"
+    script "../main.nf"
+    process "OPENMS_IDMERGER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "openms"
+    tag "openms/idmerger"
+
+    test("proteomics - openms - idxml") {
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                            [ id:'test'], // meta map
+                            [ file(params.test_data['proteomics']['openms']['idxml1'], checkIfExists: true),
+                              file(params.test_data['proteomics']['openms']['idxml2'], checkIfExists: true)]
```

Comment:
New way of writing file paths now, instead of this:
file(params.test_data['proteomics']['openms']['idxml1'], checkIfExists: true)
do this:
file(params.modules_testdata_base_path + 'proteomics/openms/HepG2_rep1_small.idXML', checkIfExists: true)